src/application/services/themeService.ts

```diff
@@ -0,0 +1,37 @@
+import { ref } from 'vue';
+
+const currentBaseTheme = ref((localStorage.getItem('baseTheme') ?? '') || 'theme-base-pure');
+const currentAccentTheme = ref((localStorage.getItem('accentTheme') ?? '') || 'theme-accent-classic');
+/**
+ * Changes Accent theme
+ *
+ * @param accentThemeName - the name of Accent theme
+ */
+const setAccentTheme = (accentThemeName:string):void => {
+  currentAccentTheme.value = accentThemeName;
```

Comment:
I think you can add relevant css class to body element here in setter function instead of watching changes in App.vue